### DISCUSSION TITLE
Filter for carbonised samples

### DISCUSSION THREAD

**Discussion by johanvonboer** [humlab-sead/sead_change_control#284] [General]
Requested by the plant macrofossil-department (Sofi).

**Comment by roger-mahler** [comment]
I need info on filter criterias and tables involved.

**Comment by roger-mahler** [comment]
Bumping issue and transferring to SEAD change control since new facets doesn't affect SEAD Query API functionality.

**Comment by roger-mahler** [comment]
We do have a filter which can be used to filter out "Carbonised" records:

![image](https://github.com/humlab-sead/sead_change_control/assets/17061116/82511645-b6fd-4490-8088-9f31aa6a65bf)

**Comment by roger-mahler** [comment]
I guess we also could setup a copy of this filter and call it "Preservation type" instead.
